package.json

```diff
@@ -30,5 +31,13 @@
     "husky": "^0.14.0",
     "nodemon": "^1.12.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-module-resolver": "^3.0.0",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
+    "babel-preset-env": "^1.6.1",
+    "flow-bin": "^0.59.0",
+    "flow-watch": "^1.1.1"
+  }
```

Comment:
So dev-deps are for local dev, these are for build deps.
That way in CI we can run yarn --production to save a couple of seconds.